src/include/executor/nodeShareInputScan.h

```diff
@@ -30,6 +30,8 @@
 #define NODESHAREINPUTSCAN_H
 
 #include "nodes/execnodes.h"
+
+#define MAX_RETRY_TIMES 5
```

Comment:
I think writer_wait_for_acks is not a must one. and for orca is on, writer_wait_for_acks is not called at all.